### ISSUE TITLE
API docs for `twisted.internet.stdio.StandardIO` missing

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4222]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4222|
|Type|defect|
|Created|2010-01-15 13:40:11Z|

http://twistedmatrix.com/documents/9.0.0/api/twisted.internet.stdio.html isn't very helpful.  If the module had an `__all__` including `'StandardIO'` things would probably be better.

Attachments:

* [stdio.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3fd/3fda26f8cff4123ddcad0c1bc89ed1e79977acef/118b2aa6f4398a1cbbdb27bf7063079aa2fa4b48.patch) (655 bytes) - added by exarkun on 2010-01-26 23:06:30Z - It was really easy

<details><summary>Searchable metadata</summary>

```
trac-id__4222 4222
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1263562811000000 1263562811000000
changetime__1264617068000000 1264617068000000
version__None None
owner__ 
cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn set owner to @exarkun|
|-|-|

This looks fine to me.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [28077]) Apply stdio.patch to explicitly expose `StandardIO` in `twisted.internet.stdio`

Author: exarkun
Reviewer: khorn
Fixes: [#4222](https://github.com/twisted/twisted/issues/4222)

Add an `__all__` to `twisted.internet.stdio` which includes `StandardIO` to make
it clear (to people and to automated tools) that `StandardIO` is the API that is
exported by this module.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
